SEED-platform/seed
# 3301 show units on column settings

#### Any background context you want to provide?
The inventory list and detail display their various columns and values. Only columns with data types of 'area' and 'eui' were appended with a unit, e.g. "Gross Floor Area (ft²)". Other columns were displayed without columns despite having hardcoded unit values defined on the PropertyState model.

#### What's this PR do?
Using the units defined on the PropertyState model the inventory list and detail columns have been updated to append units if they exist.

The original functionality has been left in to allow for backward compatibility with files in `/views`

The following columns will be appended with their units: 
```
        'gross_floor_area': 'ft²',
        'conditioned_floor_area': 'ft²',
        'occupied_floor_area': 'ft²',
        'site_eui': 'kBtu/ft²/year',
        'site_eui_weather_normalized': 'kBtu/ft²/year',
        'site_eui_modeled': 'kBtu/ft²/year',
        'source_eui': 'kBtu/ft²/year',
        'source_eui_weather_normalized': 'kBtu/ft²/year',
        'source_eui_modeled': 'kBtu/ft²/year',
        'total_ghg_emissions': 'MtCO2e/year',
        'total_marginal_ghg_emissions': 'MtCO2e/year',
        'total_ghg_emissions_intensity': 'kgCO2e/ft²/year',
        'total_marginal_ghg_emissions_intensity': 'kgCO2e/ft²/year',
```

#### How should this be manually tested?
Upload a spreadsheet that contains these columns (and others) and confirm that the column headers on the inventory list/legacy/detail contain units 


#### What are the relevant tickets?
#3301

#### Screenshots (if appropriate)
![Screen Shot 2022-07-19 at 12 43 56 PM](https://user-images.githubusercontent.com/58446472/179825486-dddb0faf-28f4-460b-9f67-3b73960687f1.png)
![Screen Shot 2022-07-19 at 12 43 17 PM](https://user-images.githubusercontent.com/58446472/179825491-4a0cbed9-f450-4887-a63f-ffbced7ffacd.png)
